### DISCUSSION TITLE
[Feature Request] Remove the new Premium button "Play next in queue"

### DISCUSSION THREAD

**Discussion by eduedp** [qnblackcat/uYouPlus#1126] [Ideas]
Youtube added a new button right on top in the three dots menu "Play next in queue" that upon click you are redirected to a page offering YT Premium features. This is very annoying because this new button is exactly where "Save to Watch later" was before. Now I always misclick this shit.

TLDR: Is it possible to remove or reorder this new button "Play next in queue" ?

Thanks!

**Comment by lyrashmi** [comment]
Yes please

**Comment by Hazem42** [comment]
Even better, is it possible to unlock this feature to be available without premium?
